### PR TITLE
Fix error response from server

### PR DESCRIPTION
**Pull request by ruchika-jain**
This PR gives out appropriate error message from server instead of a generic response having only the error response code. 

Previous: 
image

Now: 
image

### CODE REVIEW

File: src/api-client.ts [hunk 1]
```diff
@@ -38,35 +42,49 @@ export async function publishOciArtifact(repository: string, releaseId: string,
...
     let errorMessage = `Failed to create package (status: ${error.response.status}) with semver ${semver}. `
-    if (error.response.status === 400) {
-      if (error.message) {
-        errorMessage += `\nResponded with: "${error.message}"`
+    switch (error.response.status) {
+      case 400: {
+        if (error.response.data.message) {
+          errorMessage += `\nResponded with: "${error.response.data.message}"`
+        }
+        break
```
Comment (srahul3) [inline src/api-client.ts:58]: The code is redundant. I see it in all cases of switch.

**Review by srahul3 — COMMENTED** [review]
Minor comment!

File: src/api-client.ts [hunk 2]
```diff
@@ -38,35 +42,45 @@ export async function publishOciArtifact(repository: string, releaseId: string,
...
 // Respond with the appropriate error message based on response
 function errorResponseHandling(error: any, semver: string): void {
-  if(error.response) {
+  if (error.response) {
     let errorMessage = `Failed to create package (status: ${error.response.status}) with semver ${semver}. `
-    if (error.response.status === 400) {
-      if (error.message) {
-        errorMessage += `\nResponded with: "${error.message}"`
+    switch (error.response.status) {
+      case 400: {
```
[the remark this answers is not in the document]
Comment (ruchika-jain) [reply src/api-client.ts]: Addressed in recent commits 👍

**Review by srahul3 — APPROVED** [review]
LGTM ✅

**Pull request merged**